discord/client.py

```diff
@@ -826,6 +867,27 @@ def get_user(self, id: int, /) -> Optional[User]:
             The user or ``None`` if not found.
         """
         return self._connection.get_user(id)
+    
+    async def try_getting_user(self, id: int) -> User:
```

Comment:
- `self.get_user` should work as well
- `self.fetch_user` is probably better than `self.http.get_user` + `User`
- Let's name this method as `getch_user`